content/en/monitors/service_level_objectives/error_budget.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Error Budget Monitors
+kind: documentation
+description: "Use Monitors to define the Service Level Objective"
+beta: true
+---
+
+<div class="alert alert-warning">
+This feature is in closed beta. Email <a href="mailto:slo-help@datadoghq.com">slo-help@datadoghq.com</a> to request access, to ask questions, or to provide feedback on this feature.
+</div>
+
+## Overview
+
+SLO error budget monitors are threshold based and notify you when a certain percentage of your SLO’s error budget has been consumed. 
+
+**Note:** Error budget monitors are only available for [metric-based SLOs][1].
+
+Example: Alert me if 75% of the error budget for my 7-day target is consumed. Warn me if 50% is consumed (optional).
+
+## Monitor creation
+
+1. Navigate to the [SLO status page][2].
+2. Create a new metric-based SLO or edit an existing one. For existing SLOs, you can also click the “Enable Alerts” link in the SLO detail side panel to take you directly to the alert configuration.
+3. Set an alert to trigger when the percentage of the error budget consumed is above the `threshold`
+over the past `target` number of days
+4. Add [Notification information][3] into the **Say what’s happening** and **Notify your team** sections
+5. Click the ‘Save and Set Alert’ button on the SLO Configuration Page.
+
+**Note:** Clicking the `New Condition` button adds an optional warning condition. The warning threshold must be less than the alert threshold.
+
+{{< img src="monitors/service_level_objectives/error_budget_alert.png" alt="setting up an error budget alert">}}
+
+### API
+
+You can create SLO error budget monitors using the create-monitor API endpoint. Below is an example query for an SLO monitor, which alerts when more than 75% of the error budget of an SLO is consumed:
```

Comment:
'create-monitor API endpoint' should link to https://docs.datadoghq.com/api/v1/monitors/#create-a-monitor